doc/languages-frameworks/rust.section.md

```diff
@@ -75,6 +75,72 @@ pkgs.rustPlatform.buildRustPackage {
 }
 ```
 
+### Running package tests
+
+When using `buildRustPackage`, the `checkPhase` is enabled by default and runs
+`cargo test` on the package to build. To make sure that we don't compile the
+sources twice and to actually test the artifacts that will be used after that, 
+the tests will be ran in the `release`-mode by default.
```

Comment:
I think "to actually test the artifacts that will be used" is a bit unnecessary. What else would `cargo test` be testing?

```suggestion
When using `buildRustPackage`, the `checkPhase` is enabled by default and runs
`cargo test` on the package. In order to prevent compiling the source twice, the
tests will be ran in `release` mode by default.
```

Reply:
> I think "to actually test the artifacts that will be used" is a bit unnecessary. What else would cargo test be testing? 

The build artifacts from the `debug` mode (i.e. without optimizations etc.).